src/test/java/com/wdtinc/mapbox_vector_tile/build/MvtBuildTest.java

```diff
@@ -244,8 +253,7 @@ private static Polygon buildPolygon(Random random, int pointCount, GeometryFacto
     }
 
     private Point createPoint() {
-        Coordinate coord = new Coordinate( (int) (RANDOM.nextDouble() * 4095),
-                (int) (RANDOM.nextDouble() * 4095));
+        Coordinate coord = new Coordinate(RANDOM.nextInt(4096), RANDOM.nextInt(4096));
```

Comment:
Is Android missing nextDouble()?

Reply:
It addressed a code quality warning.  Findbugs warns that the previous code was slow because of the cast...vs asking immediately for an int.